package.json

```diff
@@ -21,16 +21,15 @@
 		"ci:fix": "biome check --write",
 		"prepack": "npm run build",
 		"postpack": "npm run clean",
-		"build": "rspack build",
+		"build": "esbuild --bundle --platform=node --target=node18 --outdir=dist src/index.ts",
```

Comment:
main reason to switch from rspack to esbuild?

Reply:
Ehm we couldnt make it work with the way rspack was bundling and re exporting the `load`  function. Maybe it could have been configured, but it was faster to switch